src/components/UpcomingEvents.jsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react'
-import { Link } from 'gatsby'
 
 const UpcomingEvents = () => {
   return (
     <section>
-      <h1>No Upcoming Events</h1>
+      <h3>No Upcoming Events</h3>
       <p>
-        Join our <Link to="">Meeting group</Link> to stay up to date on what's
-        coming up.
+        Join our{' '}
+        <a href="https://www.meetup.com/code-for-chicago/" target="_blank">
```

Comment:
Let's add `rel="noreferrer"` to this a tag. There is a lot of arguments that this is unnecessary, but let's be safe rather than sorry. To learn more here's a brief article on the topic. https://www.reliablesoft.net/noreferrer-noopener/